### ISSUE TITLE
Quest: Parts of the Swarm (Nighthaven)

### ISSUE THREAD

**Issue by Firestoat** [elysium-project/bug-tracker#55]
Quest link -- http://www.wowhead.com/quest=1148/parts-of-the-swarm
Extra link -- http://www.wowdb.com/quests/1148-parts-of-the-swarm

Issue - Back on old retail in 2005, this quest could be obtained at level 28 which was perfect for my cow Shaman to use in WSG at the time. I noticed this quest is incorrectly set to level 31. Above is the depreciated link to WoWhead confirming the level of 28 required, along with an old Curse gaming link for that level as well.

**Comment by Terranzin** [comment]
I´ve reviewed your issue. 

The item does say "Requires Level 28", this isnt correct. The quest should according to the archieve start at level 29, this means that the text needs to be changed for the item Cracked Silithis Carapace (ID: 5877)
![image](https://user-images.githubusercontent.com/25831382/32968900-38dd81de-cbe3-11e7-9a6c-9a90ce8d12e3.png)

Also another issue is that the quest does not start at 28, as you reported. But you can start it at level 30, but this isnt correct either, since the quest requirements should be met at level 29.

Last of all, level 31 as you state is the "recommended" level for this quest.

TLDR; 
- Text needs to be changed from level 28 to level 29 on itemID: 5877
- Quest requirement needs to be changed from level 30 to level 29 (Quest id: 1148)

**Comment by Firestoat** [comment]
Thank you for taking a look. I knew for certain that I was able to have my cow shaman on NA Twisting Nether server use the axe reward in the 29 WSG. It wasn't the best thing out there but it was better than what a lot of 29's were using.
